src/uint256.rs

```diff
@@ -456,3 +483,53 @@ fn check_from_str_radix_overflow() {
     let val = Uint256::from_str_radix(super_huge, 10);
     assert!(val.is_err())
 }
+
```

Comment:
These are very similar to the tests in int256, notably lacking any checks on negative values